http-delegation/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This allows us to ensure the two Gradle support projects are compliant with the Copyright/License header.